### ISSUE TITLE
CSS or Font request

### ISSUE THREAD

**Issue by Bboy486** [MattsShack/Plex-Movie-Poster-Display#16]
Feature Request to be able to use a Google Fonts (since they work with any web device) or custom fonts like movie fonts here https://famfonts.com/movies

**Comment by ZeroCoolGOS** [comment]
@Bboy486  @MattsShack 

A prototype of this feature is now in development.

**Comment by ZeroCoolGOS** [comment]
@MattsShack 

I have a working prototype almost ready for review.  Would you like to include any fonts from the famFonts site?  If so what fonts would you like us to embed?

I have not yet setup to support external fonts (ref from diff sites) but we can upload 'stock' (embeded) or 'custom' ttf files.

**Comment by ZeroCoolGOS** [comment]
@Bboy486 

This feature has been merged into the latest dev branch.

Please give a try and let us know if that works for your needs.

**Comment by Bboy486** [comment]
> @Bboy486 
> 
> This feature has been merged into the latest dev branch.
> 
> Please give a try and let us know if that works for your needs.

I updated the dev branch. When I tick custom font it only shows none from the drop-down. It looks like you are requiring downloading fonts but my request was to use css with the font-family attribute to pull in google fonts. 
Ex.
<head>
<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Audiowide|Sofia|Trirong">
<style>
h1.a {font-family: "Audiowide", sans-serif;}
h1.b {font-family: "Sofia", sans-serif;}
h1.c {font-family: "Trirong", serif;}
</style>
</head>

![Screenshot_20210206-191012_Chrome.jpg](https://user-images.githubusercontent.com/16495657/107134416-107b1780-68af-11eb-98e6-2fa651a1fd3b.jpg)

**Comment by ZeroCoolGOS** [comment]
> > @Bboy486 
> > 
> > This feature has been merged into the latest dev branch.
> > 
> > Please give a try and let us know if that works for your needs.
> 
> I updated the dev branch. When I tick custom font it only shows none from the drop-down. It looks like you are requiring downloading fonts but my request was to use css with the font-family attribute to pull in google fonts. 
> Ex.
> <head>
> <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Audiowide|Sofia|Trirong">
> <style>
> h1.a {font-family: "Audiowide", sans-serif;}
> h1.b {font-family: "Sofia", sans-serif;}
> h1.c {font-family: "Trirong", serif;}
> </style>
> </head>
> 
> ![Screenshot_20210206-191012_Chrome.jpg](https://user-images.githubusercontent.com/16495657/107134416-107b1780-68af-11eb-98e6-2fa651a1fd3b.jpg)

That is correct. You will (currently) need to download your fronts that you would like to see.  I have yet to find a way to programmatically get a list of the google fonts, and there would be so many to put in the drop down.

To keep it simple for now, and not be overwhelmed with hundreds of fonts. You can download the fonts you like from google fonts and upload to your server. 

I have tested with downloading some fonts from google fonts and the other font page suggested.  Just uploading the zip files from the sites should do the trick. 

Also this is our first attempt at allowing users to change the fonts.  We will look at expanding the functionality in the future as there are a few other things that could also be addressed in the back end.

**Comment by Bboy486** [comment]
Understood. Might be a way with css and php as you would do with a Drupal site. 

Also this might be something
https://github.com/google/fonts
